phpBB/phpbb/db/migration/data/v32x/add_missing_config.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v32x;
+
+use phpbb\db\migration\migration_interface;
+use phpbb\db\migrator;
+
+class add_missing_config extends \phpbb\db\migration\container_aware_migration
+{
+	static public function depends_on()
+	{
+		return [
+			'\phpbb\db\migration\data\v32x\v329',
```

Comment:
As this will be merged after 3.2.10-RC1, I think this should depend on v3210rc1

Reply:
This would result in a cyclic dependency: `add_missing_config` depends on `v3210rc1` depends on `font_awesome_update_cdn` depends on `add_missing_config`

Reply:
Ah ok, understood.